src/apos/components/ApostropheTiptapImportData.vue

```diff
@@ -36,7 +36,7 @@
 </template>
 
 <script>
-import XLSX from 'xlsx';
+import { read, utils } from 'xlsx';
```

Comment:
ℹ️ fix webpack module "`xlsx__WEBPACK_IMPORTED_MODULE_0__["default"]` is undefined" error